Do brownie tests work?

# Description

I'm trying to execute the brownie tests as described [here](https://github.com/liquity/dev#brownie-tests) and it seems it's throwing an error due to a missing parameter. Once I add it I get a different error.

```bash
        # LQTY
        contracts.lqtyStaking = LQTYStaking.deploy({ 'from': accounts[0] })
        contracts.communityIssuance = CommunityIssuance.deploy({ 'from': accounts[0] })
        contracts.lockupContractFactory = LockupContractFactory.deploy({ 'from': accounts[0] })
>       contracts.lqtyToken = LQTYToken.deploy(
            contracts.communityIssuance.address,
            contracts.lqtyStaking.address,
            contracts.lockupContractFactory.address,
            accounts[0], # bountyAddress
            accounts[0],  # lpRewardsAddress
            { 'from': accounts[0] }
        )
E       ValueError: constructor Sequence has incorrect length, expected 6 but got 5
```
## Expected bahavior

To execute `brownie test -s` and get a successful execution.

--- a/packages/contracts/brownie-config.yaml
+++ b/packages/contracts/brownie-config.yaml
@@ -7,7 +7,7 @@ networks:
         default_contract_owner: true
         cmd_settings:
             port: 8545
-            gas_limit: 6721975
+            gas_limit: 8000000
             accounts: 1000
             chain_id: 1337
             network_id: 1588949648

--- a/packages/contracts/tests/simulation_test.py
+++ b/packages/contracts/tests/simulation_test.py
@@ -137,6 +137,7 @@ def contracts():
         contracts.lockupContractFactory.address,
         accounts[0], # bountyAddress
         accounts[0],  # lpRewardsAddress
+        accounts[0],  # multisigAddress
         { 'from': accounts[0] }
     )
 

